### ISSUE TITLE
4.3.0 broken

### ISSUE THREAD

**Issue by saknarak** [socketio/socket.io-client#1505]
**4.3.0 is broken**
`io` is not a function

**To Reproduce**
```sh
npm init -y
npm install socket.io-client
node index
```js
// index.js
const io = require('socket.io-client')
const socket = io('http://localhost:7000', {})
```
## error output
```sh
const socket = io('http://localhost:7000')
               ^

TypeError: io is not a function
    at Object.<anonymous> (D:\work\sak\test-socket\index.js:2:16)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47
```
## Package version
```
  "dependencies": {
    "socket.io-client": "^4.3.0"
  }
```

**Expected behavior**
`io` should be a function

**Comment by darrachequesne** [comment]
Hi! The following should work:

```js
const { io } = require('socket.io-client')
const socket = io('http://localhost:7000', {})
```

That is a breaking change though, I'm digging into this :eyes:

**Comment by saknarak** [comment]
Thank you!

**Comment by darrachequesne** [comment]
This should be fixed in version 4.3.1, sorry for this.
